app/main.py

```diff
@@ -1 +1,11 @@
 # write your code here
+
+
+def copy_file(command: str) -> None:
+    _, src_file, dest_file = command.split()
```

Comment:
Validate that command.split have exactly 3 parameters
Also need to check if command if cp